std/portable/index.js

```diff
@@ -80,7 +80,9 @@ Object.defineProperties(
     "MAX_VALUE": { value: Math.fround(3.4028235e+38), writable: false },
     "MIN_NORMAL_VALUE":  { value:  Math.fround(1.17549435e-38), writable: false },
     "MIN_SAFE_INTEGER":  { value: -16777215, writable: false },
-    "MAX_SAFE_INTEGER":  { value:  16777215, writable: false }
+    "MAX_SAFE_INTEGER":  { value:  16777215, writable: false },
+    "POSITIVE_INFINITY": { value:  Math.fround(Infinity), writable: false },
+    "NEGATIVE_INFINITY": { value: Math.fround(-Infinity), writable: false }
```

Comment:
The `Math.fround(+/-Infinity)` here looks like it has no effect in JS due to the final value effectively being a `number` anyway.

Reply:
Makes me wonder about the ones above, but these may actually round to something different than the input? Or can we precompute these somehow?

Reply:
Makes sense. I precomputed all this values

Reply:
```
> Math.fround(1.1920929e-07) == 1.1920928955078125e-07
true
> Math.fround(1.4012985e-45) == 1.4012984643248170e-45
true
> Math.fround(3.4028235e+38) == 3.4028234663852886e+38
true
> Math.fround(1.17549435e-38) == 1.1754943508222875e-38
true
> Math.fround(Infinity) == Infinity
true
> Math.fround(-Infinity) == -Infinity
true
```
👍